src/middleware/externalRequests.js

```diff
@@ -1,8 +1,22 @@
 'use strict'
 
 const axios = require('axios')
-
+const fs = require('fs')
+const path = require('path')
 const logger = require('../logger')
+const {createOrchestration} = require('../orchestrations')
+
+let mediatorConfigJson, readError
+
+try {
+  const mediatorConfigFile = fs.readFileSync(
+    path.resolve(__dirname, '..', '..', 'mediatorConfig.json')
+  )
+  mediatorConfigJson = JSON.parse(mediatorConfigFile)
+} catch (err) {
+  readError = err.message
+  logger.error(`Mediator config file could not be retrieved: ${err.message}`)
+}
```

Comment:
@bradsawadye This logic already exists in the openhim.js file. Is it possible to move this logic there instead of re-reading the mediator.json file? This filesystem interaction for the urn field seems overkill

Reply:
done in https://github.com/jembi/openhim-mediator-mapping/pull/36/commits/e671af21b1ef83c3828dbd0adf3e044160ca9074